amoro-ams/amoro-ams-dashboard/src/views/catalogs/Detail.vue

```diff
@@ -533,9 +535,15 @@ async function deleteCatalogModal() {
   Modal.confirm({
     title: t('deleteCatalogModalTitle'),
     onOk: async () => {
-      await delCatalog(formState.catalog.name || '')
-      message.success(`${t('remove')} ${t('success')}`)
-      emit('updateEdit', false)
+      try {
+        await delCatalog(formState.catalog.name || '')
+        message.success(`${t('remove')} ${t('success')}`, 1, () => {
+          router.replace({ path: '/catalogs', query: {} }).then(() => router.go(0))
```

Comment:
Is this operation to refresh the page?

Reply:
> Is this operation to refresh the page?

Yes

Reply:
Calling [getConfigInfo](https://github.com/apache/amoro/blob/master/amoro-ams/amoro-ams-dashboard/src/views/catalogs/Detail.vue#L233C16-L233C29) function again maybe also refresh the data. Can you have a try?

Reply:
I guess the catalog list should be refreshed after deleting a catalog, so is there an event (like delete event) to tell the `index.vue` component to refresh the list? If my inference is wrong, please correct me. Thanks!